net-im/err/err-6.1.7.ebuild

```diff
@@ -44,9 +45,8 @@ RDEPEND="${DEPEND}
 		dev-python/pyasn1[${PYTHON_USEDEP}]
 		dev-python/pyasn1-modules[${PYTHON_USEDEP}]
 		dev-python/slixmpp[${PYTHON_USEDEP}]
-	)
-"
-S="${WORKDIR}/${MY_P}"
+	)"
+BDEPEND="test? ( dev-python/mock[${PYTHON_USEDEP}] )"
 
 distutils_enable_tests pytest
```

Comment:
btw, I suspect the IRC comment below is obsolete now

Reply:
Oh, yeah, it's a leftover from old version, I'll remove it